tests/outputs/a_expense/expenses.yaml

```diff
@@ -1,95 +1,103 @@
 2003sb_google:
-	begin_day: 22
-	begin_month: 3
-	begin_year: 2020
-	end_day: 22
-	end_month: 3
-	end_year: 2020
 	expense_type: business
-	grant_percentages:
-		- 100
+	grant_percenatges:
+		- 50.0
+		- 50.0
 	grants:
 		- dmref15
+		- mrsec14
 	itemized_expenses:
-		- day: 22
-		  month: 3
-		  purpose: payment for cloud services
-		  segregated_expense: 0.0
-		  unsegregated_expense: 0.0
-	notes: []
-	overall_purpose: Payment to Google Cloud to pay for cloud services
+		- day: 18
+		- month: june
+		- year: 2020
+		- purpose: Payment to Google Cloud to pay for cloud services
 	payee: sbillinge
-	reimbursments:
-		- amount: 159.18
-		  day: 6
-		  month: 4
-		  submission_day: 22
-		  submission_month: 3
-		  submission_year: 2020
-		  where: Bank
-		  year: 2020
-	status: submitted
-2003st_beamtime1:
-	begin_day: 14
-	begin_month: Feb
-	begin_year: 2020
-	end_day: 18
-	end_month: Feb
-	end_year: 2020
-	expense_type: travel
+	project: google
+	overall_purpose: Payment to Google Cloud to pay for cloud services
+test:
+	expense_type: business
 	grant_percentages:
-		- 100
+		- 50
+		- 50
 	grants:
+		- dmref15
 		- mrsec14
 	itemized_expenses:
-		- day: 14
-		  month: Feb
-		  year: 2020
-		  purpose: Buy the train ticket (Penn station to Ronkonkoma LIRR station)
-		  unsegregated_expense: 14.25
-		  segregated_expense: 0.0
-		- day: 14
-		  month: Feb
-		  year: 2020
-		  purpose: Pay the rent of the dorm at BNL for three nights
-		  unsegregated_expense: 156.0
-		  segregated_expense: 0.0
-		- day: 16
-		  month: Feb
-		  year: 2020
-		  purpose: Buy lunch for three group members (Rohan Pokratath, Sani Harouna-Mayer,
-			  Hung Vuong) and myself from TGI Friday's
-		  unsegregated_expense: 82.42
-		  segregated_expense: 0.0
-		- day: 17
-		  month: Feb
-		  year: 2020
-		  purpose: Buy dinner for myself from TGI Friday's
-		  unsegregated_expense: 27.67
-		  segregated_expense: 0.0
-		- day: 18
-		  month: Feb
-		  year: 2020
-		  purpose: Buy lunch for myself from wasabi
-		  unsegregated_expense: 19.65
-		  segregated_expense: 0.0
-		- day: 18
-		  month: Feb
-		  year: 2020
-		  purpose: Pay for the Uber (BNL to Ronkonkoma LIRR station)
-		  unsegregated_expense: 23.41
-		  segregated_expense: 0.0
-		- day: 18
-		  month: Feb
-		  year: 2020
-		  purpose: Buy the train ticket (Ronkonkoma LIRR station to Penn station)
-		  unsegregated_expense: 14.25
-		  segregated_expense: 0.0
-	notes: []
-	overall_purpose: Travel to the Brookhaven National Laboratory (BNL) to conduct x-ray
-		diffraction experiments
-	payee: sstao
-	status: submitted
+		- day: 1
```

Comment:
these seem to have values entered.    Also, please use dates.  Please could you maybe get together with someone to work on getting the tests as good as they can be?

Reply:
Thank you. I will reach out to Daniela

Reply:
I will reach out to Daniela.